reports/summary-2020.md

```diff
@@ -0,0 +1,814 @@
+reporting and feedback mechanism. 
+    
+#### What we've done
+
+- Changed [Community Meeting] cadence from weekly to monthly, and changed 
+meeting style from a "read out" of updates to more discussion oriented.
+- Encouraged groups to use asychronous methods for delivering updates and 
+gathering feedback. For example: Slack "standup" theads which feed into larger 
+scale reporting to a group on a mailing list.
+- Created an internal marketing group under SIG Contributor Experience to help
+  facilitate community communication.
+
+
+[Community Meeting]: https://k8s.dev/events/community-meeting/
+[community groups to report in and out]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md
+[Governance requirements]: https://git.k8s.io/community/committee-steering/governance/sig-governance.md#chair
+    
+#### Areas of research 
+
+- Review SIG charters to re-focus goals of each group at the current state of
+  maturity
+- Expand on async meeting adoption and establish best practices 
+- Improving subproject communication and connective glue back to the sponsoring
+  SIG
+- Clarify conditions for archiving or putting a subproject in maintenance mode
+- Connecting with the end user community and understanding what they want to
+  hear from upstream
+  
+    
+### Refining the Contributor Lifecycle
+
+There are a number of ways to define membership to the Kubernetes project
+and its community groups. The project's community groups must define
+their own membership terms and then various other activities and roles.
+Membership can drive:
+
+- Kubernetes GitHub Organization membership 
+- Consensus 
+- Voting
+
+**Project Membership**
+
+The Kubernetes project defines [membership] as contributors added to to one of 
+the four
+GitHub organizations. Membership grants a contributor access to test their pull
+requests, among other benefits. Membership also defines
+a member's role within the project based on their place on the [Contributor
+Ladder][ldr]. Membership does not grant voting rights for Steering Committe and
+governance matters; there is a separate set of criteria for [elections].
+
+There are two areas to address: early membership (onboarding) and sustainable 
+membership (ongoing contributor activity).
+
+**Early Membership**
+
+When new contributors onboard to the project, they often ask how to become a 
+member of a SIG. Community group membership is currently ambiguous, and the 
+options available are overwhelming for new contributors. Many SIGs reported that
+"joining the SIG mailing list" is how they define membership.
+Other responses included Slack channel membership and OWNERs files. Some groups
+requested council from the Steering Committee on defining membership.
+
+Most community groups do not have a voting process as they are driven by 
+consensus from OWNERs files and the community-at-large. However, in some cases, 
+particularly leadership transitions, community groups voted to decide.
```

Comment:
> voting process

For what? general decisions? membership? leadership?

> driven by consensus from OWNERs files

What is driven by consensus via OWNERs files?

This sentence does not have a subject and is thus grammatically incomplete (and deeply confusing). What are you talking about specifically?